src/serialization/interval_log/mod.rs

```diff
@@ -407,12 +407,12 @@ pub enum IntervalLogWriterError<E> {
     /// Histogram serialization failed.
     SerializeError(E),
     /// An i/o error occurred.
-    IoError(io::ErrorKind),
+    IoError(io::Error),
```

Comment:
Ooops, looks like I missed one: this should also override `source`.

Reply:
done